src/styles/global.css

```diff
@@ -17,3 +17,7 @@ a:hover {
 .content ul {
   @apply my-6;
 }
+
+.check-box-1:checked ~ .check-1 {
+  @apply text-opacity-100;
+}
```

Comment:
Is this code being used? Please remove it if not

Reply:
yes, it is.